2025-01-31 22:09:15,071:DEBUG:certbot._internal.main:certbot version: 3.1.0
2025-01-31 22:09:15,071:DEBUG:certbot._internal.main:Location of certbot entry point: /usr/local/bin/certbot
2025-01-31 22:09:15,071:DEBUG:certbot._internal.main:Arguments: ['--webroot', '--webroot-path=/var/www/certbot', '--email', 'karimmaroous@gmail.com', '--agree-tos', '--no-eff-email', '--staging', '-d', 'opsforall.net']
2025-01-31 22:09:15,071:DEBUG:certbot._internal.main:Discovered plugins: PluginsRegistry(PluginEntryPoint#manual,PluginEntryPoint#null,PluginEntryPoint#standalone,PluginEntryPoint#webroot)
2025-01-31 22:09:15,093:DEBUG:certbot._internal.log:Root logging level set at 30
2025-01-31 22:09:15,098:DEBUG:certbot._internal.plugins.selection:Requested authenticator webroot and installer None
2025-01-31 22:09:15,098:DEBUG:certbot._internal.plugins.selection:Single candidate plugin: * webroot
Description: Saves the necessary validation files to a .well-known/acme-challenge/ directory within the nominated webroot path. A separate HTTP server must be running and serving files from the webroot path. HTTP challenge only (wildcards not supported).
Interfaces: Authenticator, Plugin
Entry point: EntryPoint(name='webroot', value='certbot._internal.plugins.webroot:Authenticator', group='certbot.plugins')
Initialized: <certbot._internal.plugins.webroot.Authenticator object at 0x753d23beb770>
Prep: True
2025-01-31 22:09:15,099:DEBUG:certbot._internal.plugins.selection:Selected authenticator <certbot._internal.plugins.webroot.Authenticator object at 0x753d23beb770> and installer None
2025-01-31 22:09:15,099:INFO:certbot._internal.plugins.selection:Plugins selected: Authenticator webroot, Installer None
2025-01-31 22:09:15,279:DEBUG:certbot._internal.main:Picked account: <Account(RegistrationResource(body=Registration(key=None, contact=(), agreement=None, status=None, terms_of_service_agreed=None, only_return_existing=None, external_account_binding=None), uri='https://acme-staging-v02.api.letsencrypt.org/acme/acct/182784314', new_authzr_uri=None, terms_of_service=None), 5c0ea7419c19db18f607881e60bb62df, Meta(creation_dt=datetime.datetime(2025, 1, 31, 21, 18, 24, tzinfo=datetime.timezone.utc), creation_host='f5a16ff59e49', register_to_eff=None))>
2025-01-31 22:09:15,280:DEBUG:acme.client:Sending GET request to https://acme-staging-v02.api.letsencrypt.org/directory.
2025-01-31 22:09:15,286:DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): acme-staging-v02.api.letsencrypt.org:443
2025-01-31 22:09:15,466:DEBUG:urllib3.connectionpool:https://acme-staging-v02.api.letsencrypt.org:443 "GET /directory HTTP/1.1" 200 974
2025-01-31 22:09:15,467:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Fri, 31 Jan 2025 22:09:15 GMT
Content-Type: application/json
Content-Length: 974
Connection: keep-alive
Cache-Control: public, max-age=0, no-cache
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "keyChange": "https://acme-staging-v02.api.letsencrypt.org/acme/key-change",
  "meta": {
    "caaIdentities": [
      "letsencrypt.org"
    ],
    "profiles": {
      "classic": "The same profile you're accustomed to",
      "tlsserver": "https://letsencrypt.org/2025/01/09/acme-profiles/"
    },
    "termsOfService": "https://letsencrypt.org/documents/LE-SA-v1.4-April-3-2024.pdf",
    "website": "https://letsencrypt.org/docs/staging-environment/"
  },
  "newAccount": "https://acme-staging-v02.api.letsencrypt.org/acme/new-acct",
  "newNonce": "https://acme-staging-v02.api.letsencrypt.org/acme/new-nonce",
  "newOrder": "https://acme-staging-v02.api.letsencrypt.org/acme/new-order",
  "renewalInfo": "https://acme-staging-v02.api.letsencrypt.org/draft-ietf-acme-ari-03/renewalInfo",
  "revokeCert": "https://acme-staging-v02.api.letsencrypt.org/acme/revoke-cert",
  "wNSqvy2YXdE": "https://community.letsencrypt.org/t/adding-random-entries-to-the-directory/33417"
}
2025-01-31 22:09:15,468:DEBUG:certbot._internal.display.obj:Notifying user: Requesting a certificate for opsforall.net
2025-01-31 22:09:15,472:DEBUG:acme.client:Requesting fresh nonce
2025-01-31 22:09:15,473:DEBUG:acme.client:Sending HEAD request to https://acme-staging-v02.api.letsencrypt.org/acme/new-nonce.
2025-01-31 22:09:15,533:DEBUG:urllib3.connectionpool:https://acme-staging-v02.api.letsencrypt.org:443 "HEAD /acme/new-nonce HTTP/1.1" 200 0
2025-01-31 22:09:15,533:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Fri, 31 Jan 2025 22:09:15 GMT
Connection: keep-alive
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-staging-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: xMYr_xatjUXYKqYdI3CbEJ7VzzBu6szCltYkDYGwGm33LZbri4c
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800


2025-01-31 22:09:15,535:DEBUG:acme.client:Storing nonce: xMYr_xatjUXYKqYdI3CbEJ7VzzBu6szCltYkDYGwGm33LZbri4c
2025-01-31 22:09:15,536:DEBUG:acme.client:JWS payload:
b'{\n  "identifiers": [\n    {\n      "type": "dns",\n      "value": "opsforall.net"\n    }\n  ]\n}'
2025-01-31 22:09:15,542:DEBUG:acme.client:Sending POST request to https://acme-staging-v02.api.letsencrypt.org/acme/new-order:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS1zdGFnaW5nLXYwMi5hcGkubGV0c2VuY3J5cHQub3JnL2FjbWUvYWNjdC8xODI3ODQzMTQiLCAibm9uY2UiOiAieE1Zcl94YXRqVVhZS3FZZEkzQ2JFSjdWenpCdTZzekNsdFlrRFlHd0dtMzNMWmJyaTRjIiwgInVybCI6ICJodHRwczovL2FjbWUtc3RhZ2luZy12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL25ldy1vcmRlciJ9",
  "signature": "YxMmFHxv3cJT8W-Y1LElogWC1BmsSMNMeLDt9YYdksFiYtADMlmHp-U_fbc5zmXHJxMc8cz-uduvax938nJDpVEcu5F_F9LarlEwKF6wgZ4Y7nyErxRVXdCcdlHCSxwq_xH2g07G0z_-hZ5n2h1BSbRlx5fYmaKv9LIghT6oWCewYlIkHKw-k1UGoRKtS3OTftz6MlJoYC5C4ckeQs57p_LXRUF5qzSz8pbyWRneLo76lVnvNsiN99-ZhmMHErsnbfmlxBqmIhjUA1XPCxeOIe6jGi9daAPLDV8Vzgn55rlhWFlUbUhKxVvxg2GxNlWj-hDBt_-Nztkf7AnI1vF6aQ",
  "payload": "ewogICJpZGVudGlmaWVycyI6IFsKICAgIHsKICAgICAgInR5cGUiOiAiZG5zIiwKICAgICAgInZhbHVlIjogIm9wc2ZvcmFsbC5uZXQiCiAgICB9CiAgXQp9"
}
2025-01-31 22:09:15,642:DEBUG:urllib3.connectionpool:https://acme-staging-v02.api.letsencrypt.org:443 "POST /acme/new-order HTTP/1.1" 201 359
2025-01-31 22:09:15,642:DEBUG:acme.client:Received response:
HTTP 201
Server: nginx
Date: Fri, 31 Jan 2025 22:09:15 GMT
Content-Type: application/json
Content-Length: 359
Connection: keep-alive
Boulder-Requester: 182784314
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-staging-v02.api.letsencrypt.org/directory>;rel="index"
Location: https://acme-staging-v02.api.letsencrypt.org/acme/order/182784314/22333881774
Replay-Nonce: xMYr_xatxwO0091ZdzbLpUFbB26_AmaEjTDEbl_4e17EI3fzjro
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "status": "pending",
  "expires": "2025-02-07T22:09:15Z",
  "identifiers": [
    {
      "type": "dns",
      "value": "opsforall.net"
    }
  ],
  "authorizations": [
    "https://acme-staging-v02.api.letsencrypt.org/acme/authz/182784314/15902517194"
  ],
  "finalize": "https://acme-staging-v02.api.letsencrypt.org/acme/finalize/182784314/22333881774"
}
2025-01-31 22:09:15,642:DEBUG:acme.client:Storing nonce: xMYr_xatxwO0091ZdzbLpUFbB26_AmaEjTDEbl_4e17EI3fzjro
2025-01-31 22:09:15,643:DEBUG:acme.client:JWS payload:
b''
2025-01-31 22:09:15,646:DEBUG:acme.client:Sending POST request to https://acme-staging-v02.api.letsencrypt.org/acme/authz/182784314/15902517194:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS1zdGFnaW5nLXYwMi5hcGkubGV0c2VuY3J5cHQub3JnL2FjbWUvYWNjdC8xODI3ODQzMTQiLCAibm9uY2UiOiAieE1Zcl94YXR4d08wMDkxWmR6YkxwVUZiQjI2X0FtYUVqVERFYmxfNGUxN0VJM2Z6anJvIiwgInVybCI6ICJodHRwczovL2FjbWUtc3RhZ2luZy12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2F1dGh6LzE4Mjc4NDMxNC8xNTkwMjUxNzE5NCJ9",
  "signature": "4f2IqAM5RMrGK0Qmni7aVhnwjFEtXyXhKBYMLfJs98TFKEs2toebb3UR6vEGUAnduVts-MdwtQtpXMwxMENht4YDDqv9ly6ysg67WUirk76By9w8wVPRR-WLl82VSl5Aj41d7XbW8hEzj4WL-cXtP9S1_q2sjt4YydB3dZHvbPfkBR8v2AVxsbzzummtZmlsvFI6J42sOC3jO1oJCd1UruOLlr-rA4ybcgrybDxPgq9WaZEcyTMu98kTDHgeUh06zgL0P3Wjs2ar1iZX728a3ewAU8CQGIoi1OAt_OwZqynSJw8bnlXfQEp0dYEw9KJNX7z5P1Dw1KD1xqAuoKPyOA",
  "payload": ""
}
2025-01-31 22:09:15,709:DEBUG:urllib3.connectionpool:https://acme-staging-v02.api.letsencrypt.org:443 "POST /acme/authz/182784314/15902517194 HTTP/1.1" 200 839
2025-01-31 22:09:15,711:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Fri, 31 Jan 2025 22:09:15 GMT
Content-Type: application/json
Content-Length: 839
Connection: keep-alive
Boulder-Requester: 182784314
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-staging-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: xMYr_xat-cEfE-RhkyF4p5djt9lkBYQfsm29p6Z9dsQlwLE1mBM
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "opsforall.net"
  },
  "status": "pending",
  "expires": "2025-02-07T22:09:15Z",
  "challenges": [
    {
      "type": "dns-01",
      "url": "https://acme-staging-v02.api.letsencrypt.org/acme/chall/182784314/15902517194/RanBtA",
      "status": "pending",
      "token": "VmPt3apyFZysCP1OM-47RQe_hYk2Zd0CR_WPqXJ-n6k"
    },
    {
      "type": "http-01",
      "url": "https://acme-staging-v02.api.letsencrypt.org/acme/chall/182784314/15902517194/zqg85Q",
      "status": "pending",
      "token": "VmPt3apyFZysCP1OM-47RQe_hYk2Zd0CR_WPqXJ-n6k"
    },
    {
      "type": "tls-alpn-01",
      "url": "https://acme-staging-v02.api.letsencrypt.org/acme/chall/182784314/15902517194/__fKWQ",
      "status": "pending",
      "token": "VmPt3apyFZysCP1OM-47RQe_hYk2Zd0CR_WPqXJ-n6k"
    }
  ]
}
2025-01-31 22:09:15,712:DEBUG:acme.client:Storing nonce: xMYr_xat-cEfE-RhkyF4p5djt9lkBYQfsm29p6Z9dsQlwLE1mBM
2025-01-31 22:09:15,712:INFO:certbot._internal.auth_handler:Performing the following challenges:
2025-01-31 22:09:15,713:INFO:certbot._internal.auth_handler:http-01 challenge for opsforall.net
2025-01-31 22:09:15,713:INFO:certbot._internal.plugins.webroot:Using the webroot path /var/www/certbot for all unmatched domains.
2025-01-31 22:09:15,713:DEBUG:certbot._internal.plugins.webroot:Creating root challenges validation dir at /var/www/certbot/.well-known/acme-challenge
2025-01-31 22:09:15,715:DEBUG:certbot._internal.plugins.webroot:Attempting to save validation to /var/www/certbot/.well-known/acme-challenge/VmPt3apyFZysCP1OM-47RQe_hYk2Zd0CR_WPqXJ-n6k
2025-01-31 22:09:15,715:DEBUG:acme.client:JWS payload:
b'{}'
2025-01-31 22:09:15,716:DEBUG:acme.client:Sending POST request to https://acme-staging-v02.api.letsencrypt.org/acme/chall/182784314/15902517194/zqg85Q:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS1zdGFnaW5nLXYwMi5hcGkubGV0c2VuY3J5cHQub3JnL2FjbWUvYWNjdC8xODI3ODQzMTQiLCAibm9uY2UiOiAieE1Zcl94YXQtY0VmRS1SaGt5RjRwNWRqdDlsa0JZUWZzbTI5cDZaOWRzUWx3TEUxbUJNIiwgInVybCI6ICJodHRwczovL2FjbWUtc3RhZ2luZy12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2NoYWxsLzE4Mjc4NDMxNC8xNTkwMjUxNzE5NC96cWc4NVEifQ",
  "signature": "ubjvWy4x6jrwSiP_NMOt7XZFBi_paz6R0JuvIIHsaQ7Q0NakM_udlxUllvTJl0wDSGuMrcAFJU9gZfNVvy2P-MeT67hiVN_iDnE_kLl6lsHcptoke3pCthdNXdqbrLNVK3T316uhUf813maBdKYe1bkweLl7_QAuAxqJ4zMZZL21aj6Vr8B1PgPYhnfxapIDRk8Kce3Cqw3CErENXPaBKYpp46KwKmqXVB0Pp55Jym2_XSSV1HzQA60cMacOGET-cz-qyK5_LRHNSNSGaf6qU1_KUyLygX9XYHmjmMjBiZObNpbI6upvVLJA6SKKHnMMTGuPK0mnOrJSlUi0Aw3Dnw",
  "payload": "e30"
}
2025-01-31 22:09:15,789:DEBUG:urllib3.connectionpool:https://acme-staging-v02.api.letsencrypt.org:443 "POST /acme/chall/182784314/15902517194/zqg85Q HTTP/1.1" 200 201
2025-01-31 22:09:15,789:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Fri, 31 Jan 2025 22:09:15 GMT
Content-Type: application/json
Content-Length: 201
Connection: keep-alive
Boulder-Requester: 182784314
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-staging-v02.api.letsencrypt.org/directory>;rel="index", <https://acme-staging-v02.api.letsencrypt.org/acme/authz/182784314/15902517194>;rel="up"
Location: https://acme-staging-v02.api.letsencrypt.org/acme/chall/182784314/15902517194/zqg85Q
Replay-Nonce: N9Ep_yiTf10kdrX79TxFB8sSH0QR3IUUymKTHMpgBsfAqD7dXb0
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "type": "http-01",
  "url": "https://acme-staging-v02.api.letsencrypt.org/acme/chall/182784314/15902517194/zqg85Q",
  "status": "pending",
  "token": "VmPt3apyFZysCP1OM-47RQe_hYk2Zd0CR_WPqXJ-n6k"
}
2025-01-31 22:09:15,789:DEBUG:acme.client:Storing nonce: N9Ep_yiTf10kdrX79TxFB8sSH0QR3IUUymKTHMpgBsfAqD7dXb0
2025-01-31 22:09:15,790:INFO:certbot._internal.auth_handler:Waiting for verification...
2025-01-31 22:09:16,790:DEBUG:acme.client:JWS payload:
b''
2025-01-31 22:09:16,792:DEBUG:acme.client:Sending POST request to https://acme-staging-v02.api.letsencrypt.org/acme/authz/182784314/15902517194:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS1zdGFnaW5nLXYwMi5hcGkubGV0c2VuY3J5cHQub3JnL2FjbWUvYWNjdC8xODI3ODQzMTQiLCAibm9uY2UiOiAiTjlFcF95aVRmMTBrZHJYNzlUeEZCOHNTSDBRUjNJVVV5bUtUSE1wZ0JzZkFxRDdkWGIwIiwgInVybCI6ICJodHRwczovL2FjbWUtc3RhZ2luZy12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2F1dGh6LzE4Mjc4NDMxNC8xNTkwMjUxNzE5NCJ9",
  "signature": "mjCuB_-zAjMriqtfhE8RvUryOSSgL6FB0PeiBJIHdfPdieJs3mj9WvO_RRPQiM7zr5P-zQQ-dZh5YuP7Iddrtr6Aez_XHBF_JAb96uR4ikshcpi8ABXXMAbcZXqBDAxRN3SLjcgha8jkEd4odxnItf9lavXfu65t6fb1rIgQmAqc8junvqcMpWhmEzc_HHd2DC086YlI0_UGySWRuX43jSVkVzhgSyA40LYsatW2u9_IRhUqiMy-qQq0vbws0FreOhr8PdmJtFDb_ZJCsspqUzuuLHpOkmktcf7nbt8dGopNKJb9c35wKMgKRW7PdfbtTjAp3QCWgLdaiYTzKs3lSQ",
  "payload": ""
}
2025-01-31 22:09:16,861:DEBUG:urllib3.connectionpool:https://acme-staging-v02.api.letsencrypt.org:443 "POST /acme/authz/182784314/15902517194 HTTP/1.1" 200 1031
2025-01-31 22:09:16,861:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Fri, 31 Jan 2025 22:09:16 GMT
Content-Type: application/json
Content-Length: 1031
Connection: keep-alive
Boulder-Requester: 182784314
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-staging-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: xMYr_xatLu6Rl7JPRmx6jCwF8daW9maQUhH0ascHQScfTF0zx9w
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "opsforall.net"
  },
  "status": "invalid",
  "expires": "2025-02-07T22:09:15Z",
  "challenges": [
    {
      "type": "http-01",
      "url": "https://acme-staging-v02.api.letsencrypt.org/acme/chall/182784314/15902517194/zqg85Q",
      "status": "invalid",
      "validated": "2025-01-31T22:09:15Z",
      "error": {
        "type": "urn:ietf:params:acme:error:connection",
        "detail": "54.226.228.6: Fetching http://opsforall.net/.well-known/acme-challenge/VmPt3apyFZysCP1OM-47RQe_hYk2Zd0CR_WPqXJ-n6k: Connection refused",
        "status": 400
      },
      "token": "VmPt3apyFZysCP1OM-47RQe_hYk2Zd0CR_WPqXJ-n6k",
      "validationRecord": [
        {
          "url": "http://opsforall.net/.well-known/acme-challenge/VmPt3apyFZysCP1OM-47RQe_hYk2Zd0CR_WPqXJ-n6k",
          "hostname": "opsforall.net",
          "port": "80",
          "addressesResolved": [
            "54.226.228.6"
          ],
          "addressUsed": "54.226.228.6"
        }
      ]
    }
  ]
}
2025-01-31 22:09:16,861:DEBUG:acme.client:Storing nonce: xMYr_xatLu6Rl7JPRmx6jCwF8daW9maQUhH0ascHQScfTF0zx9w
2025-01-31 22:09:16,862:INFO:certbot._internal.auth_handler:Challenge failed for domain opsforall.net
2025-01-31 22:09:16,862:INFO:certbot._internal.auth_handler:http-01 challenge for opsforall.net
2025-01-31 22:09:16,862:DEBUG:certbot._internal.display.obj:Notifying user: 
Certbot failed to authenticate some domains (authenticator: webroot). The Certificate Authority reported these problems:
  Domain: opsforall.net
  Type:   connection
  Detail: 54.226.228.6: Fetching http://opsforall.net/.well-known/acme-challenge/VmPt3apyFZysCP1OM-47RQe_hYk2Zd0CR_WPqXJ-n6k: Connection refused

Hint: The Certificate Authority failed to download the temporary challenge files created by Certbot. Ensure that the listed domains serve their content from the provided --webroot-path/-w and that files created there can be downloaded from the internet.

2025-01-31 22:09:16,865:DEBUG:certbot._internal.error_handler:Encountered exception:
Traceback (most recent call last):
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 108, in handle_authorizations
    self._poll_authorizations(authzrs, max_retries, max_time_mins, best_effort)
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 212, in _poll_authorizations
    raise errors.AuthorizationError('Some challenges have failed.')
certbot.errors.AuthorizationError: Some challenges have failed.

2025-01-31 22:09:16,865:DEBUG:certbot._internal.error_handler:Calling registered functions
2025-01-31 22:09:16,865:INFO:certbot._internal.auth_handler:Cleaning up challenges
2025-01-31 22:09:16,865:DEBUG:certbot._internal.plugins.webroot:Removing /var/www/certbot/.well-known/acme-challenge/VmPt3apyFZysCP1OM-47RQe_hYk2Zd0CR_WPqXJ-n6k
2025-01-31 22:09:16,865:DEBUG:certbot._internal.plugins.webroot:All challenges cleaned up
2025-01-31 22:09:16,865:DEBUG:certbot._internal.log:Exiting abnormally:
Traceback (most recent call last):
  File "/usr/local/bin/certbot", line 33, in <module>
    sys.exit(load_entry_point('certbot', 'console_scripts', 'certbot')())
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/main.py", line 19, in main
    return internal_main.main(cli_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 1873, in main
    return config.func(config, plugins)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 1579, in certonly
    lineage = _get_and_save_cert(le_client, config, domains, certname, lineage)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 142, in _get_and_save_cert
    lineage = le_client.obtain_and_enroll_certificate(domains, certname)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 518, in obtain_and_enroll_certificate
    cert, chain, key, _ = self.obtain_certificate(domains)
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 429, in obtain_certificate
    orderr = self._get_order_and_authorizations(csr.data, self.config.allow_subset_of_names)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 497, in _get_order_and_authorizations
    authzr = self.auth_handler.handle_authorizations(orderr, self.config, best_effort)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 108, in handle_authorizations
    self._poll_authorizations(authzrs, max_retries, max_time_mins, best_effort)
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 212, in _poll_authorizations
    raise errors.AuthorizationError('Some challenges have failed.')
certbot.errors.AuthorizationError: Some challenges have failed.
2025-01-31 22:09:16,871:ERROR:certbot._internal.log:Some challenges have failed.
